# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/ProducerConsumerQueue.h
# Commit: e1b0906509ef
# Full Hash: e1b0906509efa6433978b9f980ec639b7eafbc6d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix non-webgl CI tests. r=handyman
#   
#   * Revert some partial webgl+oop+vr code.
#   * More missing FuncScope.
#   * Fix compile errors.
# ==============================================================================

diff -r 7e2a2b1b416f -r e1b0906509ef dom/canvas/ProducerConsumerQueue.h
--- a/dom/canvas/ProducerConsumerQueue.h	Wed Jan 08 22:19:23 2020 +0000
+++ b/dom/canvas/ProducerConsumerQueue.h	Wed Jan 08 22:19:26 2020 +0000
@@ -1160,10 +1160,7 @@
                              : status;
   }
 
-  template <>
-  PcqStatus TryPeekRemoveHelper<>(ConsumerView&) {
-    return PcqStatus::Success;
-  }
+  PcqStatus TryPeekRemoveHelper(ConsumerView&) { return PcqStatus::Success; }
 
   // If an item is available then it is copied into aArg.  The item is skipped
   // over if aArg is null.